src/ert/config/queue_config.py

```diff
@@ -107,57 +180,26 @@ def from_dict(cls, config_dict: ConfigDict) -> QueueConfig:
             queue_options[queue_system].append(
                 (option_name, values[0] if values else "")
             )
-            if (
-                values
-                and option_name == "SUBMIT_SLEEP"
-                and selected_queue_system == queue_system
-            ):
-                submit_sleep = float(values[0])
-
-        project_code_is_set = next(
-            (
-                True
-                for key, _ in queue_options[selected_queue_system]
-                if key == "PROJECT_CODE"
-            ),
-            False,
+        queue_options[selected_queue_system] = _add_generic_queue_options(
+            config_dict, queue_options[selected_queue_system]
         )
-        if selected_queue_system != QueueSystem.LOCAL and not project_code_is_set:
+
+        if "PROJECT_CODE" not in _option_list_to_dict(
+            queue_options[selected_queue_system]
+        ):
```

Comment:
I guess PROJECT_CODE won't hurt on when using queue `LOCAL`.

Reply:
That is what I was thinking.